orchestrator-salt/src/main/resources/salt/salt/gateway/config/cm/topology_api.xml.j2

```diff
@@ -125,7 +125,7 @@
             </param>
             <param>
                 <name>{{ aclsauthz_prefix ~ "resourcemanager.acl" }}</name>
-                <value>*;{{ salt['pillar.get']('gateway:envAccessGroup') }};*</value>
```

Comment:
Why are there 3 files where we're making the resourcemanager change ?

Reply:
Updated this to only change `topology.xml.j2`, this will in turn get populated into `cdp-proxy.xml`